src/classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  attr_accessor :label, :students
```

Comment:
- [ ] **Optional:** It is conventional to have the accessors at the top of the class before the initialize method.